src/index.js

```diff
@@ -1,10 +1,94 @@
-// This is the JavaScript entry file - your code begins here
-// Do not delete or rename this file ********
-
-// An example of how you tell webpack to use a CSS (SCSS) file
 import './css/index.scss';
+import Traveler from '../src/Traveler';
+import Trip from './Trip';
+import TripsRepo from './TripRepo';
+import Destination from './Destination';
+import DestinationsRepo from './DestinationRepo'
+import apiCalls from '../src/apis'
+import domUpdates from '../src/domUpdates'
+
+const displayPastTripsElement = document.querySelector('.past-trip-js');
+const displayPresentTripsElement = document.querySelector('.present-trip-js');
+const displayUpcomingTripsElement = document.querySelector('.upcoming-trip-js');
+const displayPendingTripsElement = document.querySelector('.pending-trip-js');
+const totalCostElement = document.querySelector('.total-cost-js');
+const yearDropDown = document.querySelector('.year-drop-down-js');
+
+
+let traveler, trips, tripsRepo, destinations, destinationsRepo;
+const todaysDate = "2020/01/01"
+
+yearDropDown.addEventListener('change', getTotalCostByYear)
+
+Promise.all([apiCalls.getTravelerData(), apiCalls.getTripsData(), apiCalls.getDestinationsData()])
+  .then(data => {
+    let destArr = []
```

Comment:
I think it's generally better to name arrays as a plural word, like `destinations` - and not include the type of object in the variable name.